tsconfig.json

```diff
@@ -1,60 +1,35 @@
-/*
-  This is a "Solution Style" tsconfig.json file, and is used by editors and TypeScript’s language server to improve development experience.
-  It is not intended to be used to perform a compilation.
-
-  To learn more about this file see: https://angular.io/config/solution-tsconfig.
-*/
 {
-  "extends": "./tsconfig.base",
-  "files": [],
-  "references": [
```

Comment:
We need to validate my hypothesis but I think we don't need to reference all sub Typescript configuration file. Each packages or projects already have a reference to their tsconfig in their execution pipeline. For development, it is not necessary to know each sub tsconfig.